### ISSUE TITLE
`arguments` field for CREATE in broadcast log is `null`

### ISSUE THREAD

**Issue by adhusson** [foundry-rs/foundry#3053]
### Component

Forge

### Have you ensured that all of these are up to date?

- [X] Foundry
- [X] Foundryup

### What version of Foundry are you on?

forge 0.2.0 (20e7386 2022-08-31T00:09:48.876171Z)

### Operating System

macOS (Apple Silicon)

### Describe the bug

The `arguments` field is populated for function calls, but constructor arguments should be there as well, especially since they're the only way to distinguish multiple deploys of the same base contract.

**Comment by mds1** [comment]
+1 I've noticed this same bug

**Comment by mattsse** [comment]
was this fixed by https://github.com/foundry-rs/foundry/pull/3136 ?

**Comment by mds1** [comment]
No that was for a different field

```json
{
  // This was fixed in #3136
  "transactionType": "CREATE",
  "contractName": "MyContract",
  "function": null,
  // This is what this PR refers to
  "arguments": null
}
```
